tasks/localizationtasks.js

```diff
@@ -149,15 +144,64 @@ gulp.task('ext:localization:xliff-to-ts', function () {
 		.pipe(gulp.dest(config.paths.project.root + '/src/constants/'));
 });
 
+// Generates a package.nls.json file from localizedPackage.json.enu.xlf
+gulp.task('ext:localization:generate-eng-package.nls', function () {
+	return gulp.src([config.paths.project.localization + '/xliff/enu/localizedPackage.json.enu.xlf'], { base: '.' })
+		.pipe(through.obj(function (file, enc, callback) {
+			// convert xliff into json document
+			let dict = convertXmlToDictionary(String(file.contents), false);
+
+			var contents = ['{'];
+			var regxForReplacingQuotes = new RegExp('"', 'g');
+
+			// Get all keys and values from localizedPackage.json.enu.xlf
+			Object.keys(dict).forEach(key => {
```

Comment:
This dictionary stuff appears to be duplicated below - would suggest looking into making a helper function for this as well

Reply:
There are subtle differences between the two dictionary loops, but I'll try to implement something to consolidate it